v1/transactionmetadata/transactionmetadata.gunk

```diff
@@ -0,0 +1,1383 @@
+//                 },
+//         },
+//         SecurityDefinitions: openapiv2.SecurityDefinitions{
+//                 Security: map[string]openapiv2.SecurityScheme{
+//                         "OAuth2": openapiv2.SecurityScheme{
+//                                 Type:             openapiv2.TYPE_OAUTH2,
+//                                 Flow:             openapiv2.FLOW_ACCESS_CODE,
+//                                 Description:      "Access to transactions resources is restricted to authenticated users. The authentication process follows the Authorization Code flow.",
+//                                 AuthorizationURL: "/v1/auth",
+//                                 TokenURL:         "/v1/token",
+//                                 Scopes: openapiv2.Scopes{
+//                                         Scope: map[string]string{
+//                                                 "read":  "Grants read access",
+//                                                 "write": "Grants write access",
+//                                         },
+//                                 },
+//                         },
+//                 },
+//         },
+// }
+package transactionmetadata
+
+import (
+	"time"
+
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+
+	types "github.com/openbank/openbank/v1/types"
+)
+
+type User struct {
+	// ID is the user identifier.
+	ID string `pb:"1" json:"id"`
+
+	// Provider is the provider of the user.
+	Provider string `pb:"2" json:"provider"`
+
+	// DisplayName is the user's display name.
+	DisplayName string `pb:"3" json:"display_name"`
+}
+
+type Tag struct {
+	// ID is the tag identifier.
+	ID string `pb:"1" json:"id"`
+
+	// Value is the tag content/value.
+	Value string `pb:"2" json:"value"`
+
+	// Date is the date the tag is created/updated.
+	Date time.Time `pb:"3" json:"date"`
+
+	// User is the user information.
+	User User `pb:"4" json:"user"`
+}
+
+type Image struct {
+	// ID is the image identifier.
+	ID string `pb:"1" json:"id"`
+
+	// Label is the label of the image.
+	Label string `pb:"2" json:"label"`
+
+	// URL is the image URL.
+	URL string `pb:"3" json:"url"`
+
+	// Date is the date the image is created/updated.
+	Date time.Time `pb:"4" json:"date"`
+
+	// User is the user information.
+	User User `pb:"5" json:"user"`
+}
+
+type Comment struct {
+	// ID is the image identifier.
+	ID string `pb:"1" json:"id"`
+
+	// Value is the comment content/value.
+	Value string `pb:"2" json:"value"`
+
+	// Date is the date the tag is created/updated.
+	Date time.Time `pb:"3" json:"date"`
+
+	// User is the user information.
+	User User `pb:"4" json:"user"`
+}
+
+type Narrative struct {
+	// Narrative is the content of the narrative.
+	Narrative string `pb:"1" json:"narrative"`
+}
+
+type WhereTag struct {
+	// Latitude is the latitude geo information.
+	Latitude float64 `pb:"1" json:"latitude"`
```

Comment:
we already have Latitude and longitude here https://github.com/openbank/openbank/blob/master/v1/types/types.gunk#L251
I recommend using the existing one

Reply:
updated, thanks for the review.